Density/app/src/main/res/layout/activity_main.xml

```diff
@@ -92,7 +92,7 @@
     </com.google.android.material.appbar.AppBarLayout>
 
     <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-        android:id="@+id/swipe_refresh"
+        android:id="@+id/swipeRefresh"
```

Comment:
Not sure if there is any "official" convention for Android resource IDs, but I think the popular convention is to have underscores. But I do like camel case since we're using Kotlin Android Extensions.